### DISCUSSION TITLE
How do I specify the server dev path ?

### DISCUSSION THREAD

**Discussion by zababurinsv** [parcel-bundler/parcel#7339] [Q&A]
When i run dev server, this server run on on path 
```
http://localhost:1234
```
How can I indicate that the path would be 
```
http://localhost:1234/some-path
```

This is important to me because I use web worker and service worker

I can change path for all file in express server, but then run workers

```
   let url = new URL('./worker.mjs', import.meta.url)
``
Its path is calculated from the site root

**Comment by devongovett** [comment]
`--public-url` - https://parceljs.org/features/cli/#parameters

**Reply by zababurinsv** [reply]
I pointed it out.
It does not work for me for dev server .

```
    "dev": {
      "includeNodeModules": true,
      "optimize": true,
      "sourceMap": false,
      "publicUrl": "/newkind-service",
      "scopeHoist": true,
      "distDir": "./service",
      "context": "browser",
      "source": "./frontend/src/index.html"
    }

 "dev": "rimraf ./service/* && parcel --cache-dir ./.parcel-cache --no-hmr --port $npm_package_config_port_dev --dist-dir service/newkind-service --target dev",
```

if i add --public-url /newkind-service

It is also not work

```
Server running at http://localhost:4517
```

But i need 
```
Server running at http://localhost:4517/newkind-service
```

PublicUrl work when i create build

In build path is true

But worker run is dynamic and in online it is bundle working, and dev not working
